templates/search.twig

```diff
@@ -297,26 +297,31 @@
 							</ul>
 							{% if ( load_more ) %}
 								<div class="col-lg-12 mb-5 load-more-button-div">
-									<button
-										class="btn btn-medium btn-small btn-secondary more-btn {{ load_more.async ? 'btn-load-more-async' : '' }} btn-load-more-click-scroll"
-										data-current_page="{{ current_page }}"
-										data-total_posts="{{ found_posts }}"
-										data-posts_per_load="{{ load_more.posts_per_load }}"
-										data-archives_per_load="{{ load_more.archives_per_load }}"
-										data-page_archive_available_after = "{{ load_more.page_archive_available_after }}">
-											{{ load_more.button_text }}
-									</button>
-
-									{% if settings.include_archive_content %}
-										<div class="toggle-wrapper">
-											<div class="toggle-button">
-												<input type="checkbox" id="show-archive-toggle" class="toggle show-archive-toggle"/>
-												<label class="toggle-label" for="show-archive-toggle"></label>
-											</div>
-											<span class="toggle-label-include">{{ load_more.include_archive_content_text }}</span>
-											<span class="toggle-label-exclude">{{ load_more.exclude_archive_content_text }}</span>
+									<div class="row">
+										<div class="col-lg-5 col-sm-12">
+											<button
+												class="btn btn-medium btn-small btn-secondary more-btn {{ load_more.async ? 'btn-load-more-async' : '' }} btn-load-more-click-scroll"
+												data-current_page="{{ current_page }}"
+												data-total_posts="{{ found_posts }}"
+												data-posts_per_load="{{ load_more.posts_per_load }}"
+												data-archives_per_load="{{ load_more.archives_per_load }}"
+												data-page_archive_available_after = "{{ load_more.page_archive_available_after }}">
+													{{ load_more.button_text }}
+											</button>
 										</div>
-									{% endif %}
+										<div class="col-lg-7 col-sm-12">
+											{# {% if true or settings.include_archive_content %} #}
```

Comment:
I think this needs to be uncommented and 'true' removed.

Reply:
Done!